radix-common/src/math/precise_decimal.rs

```diff
@@ -27,15 +27,20 @@ use crate::*;
 /// The finite set of values are of the form `m / 10^36`, where `m` is
 /// an integer such that `-2^(256 - 1) <= m < 2^(256 - 1)`.
 ///
+/// ```text
 /// Fractional part: ~120 bits / 36 digits
 /// Integer part   : 136 bits / 41 digits
 /// Max            :  57896044618658097711785492504343953926634.992332820282019728792003956564819967
 /// Min            : -57896044618658097711785492504343953926634.992332820282019728792003956564819968
+/// ```
 ///
-/// Unless otherwise specified, all operations will panic if underflow/overflow.
+/// Unless otherwise specified, all operations will panic if there is underflow/overflow.
+///
+/// To create a PreciseDecimal with a certain number of precise `10^(-36)` subunits,
+/// use [`PreciseDecimal::from_precise_subunits`].
 #[cfg_attr(feature = "fuzzing", derive(Arbitrary))]
 #[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct PreciseDecimal(pub InnerPreciseDecimal);
```

Comment:
Removing `pub` might be painful for dApp devs. I've seen you asked in the devs channel if ok to do this. They agreed but not sure if every dev got to read it.